apps/search/style/search.css

```diff
@@ -61,14 +56,16 @@ body, a {
 #suggestions-provider {
   color: #00aac5;
   position: relative;
-  padding: 0 2rem 0 1rem;
+  padding: 0;
+  -moz-padding-end: 2rem;
+  -moz-padding-start: 1rem;
 }
 
 #suggestions-provider::before {
   content: ' ';
-  position: absolute;
-  bottom: 0.2rem;
-  right: .8rem;
+  position: relative;
```

Comment:
Doing it as Ahmed did seems the right way to do it, but it didn't work for me (neither to him, because it caused the regression about the dropdown in LTR languages). I tried with relative positions, but you are right that is harder. I'm having some issues with RTL and pseudo-elements (as gmarty was pointing out in https://github.com/mozilla-b2g/gaia/pull/28866#discussion-diff-26511274L20) so I ended up using a new element.